lib/router.js

```diff
@@ -79,9 +79,10 @@ Cherrytree.prototype.map = function (routes) {
 
   function eachBranch (node, memo, fn) {
     node.routes.forEach(function (route) {
-      if (!route.routes || route.routes.length === 0) {
+      if (!route.options || route.options.abstract !== true) {
```

Comment:
What would be the use case for the `abstract` route in this new version? Ideally it should work only one way (e.g. route to leaf routes only or route to any route only).